src/main/java/domain/LottoTicketGenerator.java

```diff
@@ -0,0 +1,28 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoTicketGenerator {
+	public static LottoTicket generateRandomLottoTicket() {
+		List<LottoNumber> shuffledNumbers = getShuffledNumbers();
+		return new LottoTicket(shuffledNumbers.subList(0, LottoTicket.LOTTO_NUMBER_COUNT));
+	}
+
+	public static LottoTickets generateRandomLottoTickets(int countOfTickets) {
+		List<LottoTicket> lottoTickets = new ArrayList<>();
+
+		for (int i = 0; i < countOfTickets; i++) {
+			lottoTickets.add(LottoTicketGenerator.generateRandomLottoTicket());
+		}
+
+		return new LottoTickets(lottoTickets);
```

Comment:
아래와 같이 변경해볼 수 있을 것 같아요. 😉

```suggestion
		return Stream.generate(LottoTicketGenerator::generateRandomLottoTicket)
			.limit(countOfTickets)
			.collect(Collectors.collectingAndThen(Collectors.toList(), LottoTickets::new));
```

Reply:
반영했습니다. https://github.com/next-step/java-lotto-kakao/pull/35/commits/cacfd8dc4fc7ff246f362242b6b6a35bcb5a2b2a